### PR TITLE
linting and cleanup

### PR DESCRIPTION
**Pull request by sbrugman**
Linting using [black](https://nicedoc.io/psf/black) and several cleanups while preparing the tensor element type feature

### CODE REVIEW

**Comment by parrt** [comment]
hi. not sure about reformatting but lint stuff useful. let me look. First, i have to fix the pdf PR as it stripped the fully-qualified path ;) haha

**Comment by sbrugman** [comment]
Two other minor points:

- `matricks_old.py` may be removed?
- The `color` variable in https://github.com/parrt/tensor-sensor/blob/master/tsensor/viz.py#L453 is not used

**Comment by parrt** [comment]
Yep, already removed that file. i'll look at color.

**Comment by parrt** [comment]
Hi. Lots of `'` -> `"` type changes and other non-semantic changes. Hmm...not sure we should make a major change just for spacing, though some are definitely worthwhile.  Also some dead code it identified. i guess I'll cherry pick on this one. thanks though!!

File: tsensor/analysis.py [hunk 1]
```diff
@@ -374,9 +374,7 @@ def listener(self, frame, event, arg):
         info = inspect.getframeinfo(frame)
         filename, line = info.filename, info.lineno
         name = info.function
-
```
Comment (parrt) [inline tsensor/analysis.py:302]: actually, it could be other events like call etc... if I remember

File: tsensor/ast.py [hunk 2]
```diff
@@ -242,9 +238,7 @@ def clarify(self):
         return f"Cause: {self.op} on tensor " + " and ".join(opnd_msgs)
 
     @property
-    def optokens(
```
Comment (parrt) [inline tsensor/ast.py]: hmm...is this an old version of file?

**Comment by parrt** [comment]
Ok, i cherry picked a bunch but didn't grab the reformatting wholesale. closing...gave you credit in the commit message.

**Pull request closed without merging**